test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -81,12 +80,14 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 			}
 		}
 		await this.waitForSuccessViewPostNotice();
+
+		const snackBarNoticeLinkSelector = By.css( '.components-snackbar__content a' );
 		const url = await this.driver.findElement( snackBarNoticeLinkSelector ).getAttribute( 'href' );
 
 		if ( visit ) {
-			const snackbar = await this.driver.findElement( snackBarNoticeLinkSelector );
-			await this.driver.executeScript( 'arguments[0].click();', snackbar );
+			await driverHelper.clickWhenClickable( this.driver, snackBarNoticeLinkSelector );
```

Comment:
Instead of clicking the button using a script, let's take into account whether it's ready and rendered on screen first.

Reply:
This is worth a shot. We changed it a while back because clickWhenClickable wasn't working.